network/hostnetwork/packethandler_test.go

```diff
@@ -322,6 +324,19 @@ func (hh *mockHostHandler) AddUncheckedNode(hostID id.ID, nonce []byte, ref core
 
 }
 
+func (hh *mockHostHandler) StartAuthorize() error {
+	return nil
+}
+
+func (hh *mockHostHandler) AddUnsync(nodeID core.RecordRef,
+	roles []core.NodeRole, address string /*, publicKey *ecdsa.PublicKey*/) (chan *core.ActiveNode, error) {
```

Comment:
comments?

Reply:
we will pass publicKey as soon as we can correctly get it from certificate